share/github-backup-utils/ghe-restore-es-hookshot

```diff
@@ -30,22 +30,30 @@ fi
 
 for index in $indices; do
   if [ -z "$last_index" ] || ! [ $index \< $last_index ]; then
-    if $CLUSTER || [ -n "$configured" ]; then
-      ghe_verbose "* Restoring $index"
-      gzip -dc $GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/hookshot/$index.gz | ghe-ssh "$GHE_HOSTNAME" "/usr/local/share/enterprise/ghe-es-load-json 'http://localhost:9201/$index'" 1>&3
-    else
-      echo "$index.gz" >> $tmp_list
-    fi
+    echo "$index.gz" >> $tmp_list
   fi
 done
 
 if [ -s "$tmp_list" ]; then
+  ghe-ssh "$GHE_HOSTNAME" -- "sudo mkdir -p '$GHE_REMOTE_DATA_USER_DIR/elasticsearch-restore'" 1>&3
+  ghe-ssh "$GHE_HOSTNAME" -- "sudo chown elasticsearch:elasticsearch '$GHE_REMOTE_DATA_USER_DIR/elasticsearch-restore'" 1>&3
+
   ghe-rsync -avz --delete \
     -e "ghe-ssh -p $(ssh_port_part "$GHE_HOSTNAME")" \
     --rsync-path="sudo -u elasticsearch rsync" \
     --files-from=$tmp_list \
     "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/hookshot/" \
     "$(ssh_host_part "$GHE_HOSTNAME"):$GHE_REMOTE_DATA_USER_DIR/elasticsearch-restore/" 1>&3
+
+  if $CLUSTER || [ -n "$configured" ]; then
+    for index in $(cat $tmp_list | sed 's/\.gz$//g'); do
+      ghe_verbose "* Restoring $index"
+      echo "export PATH=\$PATH:/usr/local/share/enterprise && gzip -dc $GHE_REMOTE_DATA_USER_DIR/elasticsearch-restore/$index | ghe-es-load-json 'http://localhost:9201/$index'" |
+      ghe-ssh "$GHE_HOSTNAME" -- /bin/bash 1>&3
+    done
+  fi
+
+  ghe-ssh "$GHE_HOSTNAME" -- "sudo rm $GHE_REMOTE_DATA_USER_DIR/elasticsearch-restore/*.gz" 1>&3
```

Comment:
As before, doesn't this `rm` need to happen as part of the `if $CLUSTER || [ -n "$configured" ];` block?

Reply:
As above.